### PR TITLE
ImageMagick test clean-up

### PR DESCRIPTION
**Pull request by rwmanos**
- clean-up ALL files
- default assert timeout
- no pkcon quit
- minor syntax

Working example: http://rwmanosvm2.suse.cz/tests/2071 (folder in nautilus test is empty)

### CODE REVIEW

File: tests/x11/ImageMagick.pm [hunk 1]
```diff
@@ -29,968 +29,972 @@ use testapi;
...
     type_string "exit\n";
 
-    type_string "wget --quiet " . data_url('imagemagick/bg_script.sh') . " -O bg_script.sh \n";
+    assert_script_run "wget --quiet " . data_url('imagemagick/bg_script.sh') . " -O bg_script.sh";
     type_string "chmod +x bg_script.sh; ./bg_script.sh " . data_url('imagemagick/bg_script.sh') . " \n";
 
-    assert_screen("imagemagick_test", 10);
+    assert_screen "imagemagick_test";
     send_key "alt-f4";
     wait_idle;
```
Comment (coolo) [inline tests/x11/ImageMagick.pm]: what is the reason for all these wait_idles? unless you're having 2 needles that look exactly the same in a row, there is no reason. And these situations you better avoid in general.

**Comment by coolo** [comment]
BTW, our test takes 19 minutes. So possibly we should move this test somewhere special - I mean it's unlikely that the way SP2 was migrated has an effect on how well ImageMagick eats image 3-199

File: tests/x11/ImageMagick.pm (hunk 1 above)
Comment (rwmanos) [inline tests/x11/ImageMagick.pm]: Yes, there are needles that look exactly the same in a row. Some images are the same although they are produced with different command. I could adjust it at some point to not have this behavior but not right now.

**Comment by rwmanos** [comment]
It was initially only for SP1 (Update tests) but you asked my why the version check. I can revert it back or even remove it for now. Feel free to adjust it yourself as well.

**Comment by coolo** [comment]
No, I want it to run on SP2 - but possibly not in x11tests. Even for SP1 maintenance we only want to see ImageMagick tested once, not in every qam-minimal-full.

**Comment by rwmanos** [comment]
I looked at it but I couldn't figure a better position. Regarding qam we are planning to discuss the overall qam-* tests in order to avoid such issues and increase test coverage. Please feel free to indicate a better place for this test for SP2.

File: tests/x11/ImageMagick.pm (hunk 1 above)
Comment (rwmanos) [inline tests/x11/ImageMagick.pm]: @coolo fixed in #2059

**Pull request merged**